.github/workflows/ci.yml

```diff
@@ -9,9 +9,6 @@ on:
 jobs:
   tests:
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        node: [ 20, 24 ]
     continue-on-error: ${{ matrix.node == 24 }}
```

Comment:
Should we remove `continue-on-error: ${{ matrix.node == 24 }}` in the previous commit?

Reply:
yes, you right, fixed it
thank you!